### DISCUSSION TITLE
New version failed when debug

### DISCUSSION THREAD

**Discussion by pablodecastillo** [byuflowlab/FLOWUnsteady#177] [General]
Hello,
Last version failed when trying to debug, exactly when using cxxwrap,     bodies = fmm.genBodies(maxparticles), i tried with both versions of Julia recomended at the installation.
Thanks

**Comment by EdoAlvarezR** [comment]
Hi Pablo! Would you mind providing versions information to be able to replicate the error? My best guess is that you haven't successfully compiled FLOWExaFMM.

**Comment by pablodecastillo** [comment]
Hi Eduardo,Following the installation steps from: Installation · FLOWUnsteady

| 
| 
|  | 
Installation · FLOWUnsteady


 |

 |

 |


I tried both versions of Julia, and Cxxwrap 1.15 , 1.15,1 and 1.16thanks


    En sábado, 20 de julio de 2024, 22:59:48 CEST, Ed Alvarez ***@***.***> escribió:  
 
 


Hi Pablo! Would you mind providing versions information to be able to replicate the error?
